x-pack/plugins/observability/server/plugin.ts

```diff
@@ -70,6 +74,12 @@ interface PluginStart {
   alerting: PluginStartContract;
 }
 
+const sloRuleTypes = [
```

Comment:
Nit: this can probably just be `const ruleTypes` instead. It's not just slo rule types.